### PR TITLE
Add chinese translation

### PR DESCRIPTION
**Pull request by zilongshanren**

### CODE REVIEW

**Comment by zilongshanren** [comment]
@dabingnn  please review my PR. thanks.

**Comment by zilongshanren** [comment]
@dabingnn  求 review 啊

File: cocos/physics/CCPhysicsWorld.h [hunk 1]
```diff
@@ -98,231 +89,376 @@ class CC_DLL PhysicsWorld
...
     void setGravity(const Vect& gravity);
     
-    /**
+    /**@~english
      * Set the speed of this physics world.
      *
-     * @attention if you setAutoStep(false), this won't work.
-     * @param speed  A float number. Speed is the rate at which the simulation executes. default value is 1.0.
+     * @~chinese 
+     * 设置物理世界的速度。
```
Comment (dabingnn) [inline cocos/physics/CCPhysicsWorld.h]: It is not clear, is it the speed of the movement of the physics world?

File: cocos/ui/UIButton.h [hunk 2]
```diff
@@ -192,85 +264,122 @@ class CC_GUI_DLL Button : public Widget
...
 
-    /**
+    /**@~english
      * Sets the title's text horizontal alignment.
      *
-     * @param hAlignment see TextHAlignment
+     * @~chinese 
+     * 设置标题的文本的水平对齐方式
+     * 
+     * @param hAlignment @~english see TextHAlignment
```
Comment (dabingnn) [inline cocos/ui/UIButton.h:355]: Do we need  `@~chinese` here?

File: cocos/ui/UIButton.h [hunk 3]
```diff
@@ -192,85 +264,122 @@ class CC_GUI_DLL Button : public Widget
...
-    /**
+    /**@~english
      * Sets the title's text vertical alignment.
      *
-     * @param hAlignment see TextHAlignment.
-     * @param vAlignment see TextVAlignment.
+     * @~chinese 
+     * 设置标题的文本垂直对齐方式
+     * 
+     * @param hAlignment @~english see TextHAlignment.
```
Comment (dabingnn) [inline cocos/ui/UIButton.h:365]: and here

**Comment by dabingnn** [comment]
@andyque Comments updated, Thanks.

File: cocos/ui/UIButton.h (hunk 2 above)
Comment (zilongshanren) [inline cocos/ui/UIButton.h:355]: @dabingnn  it's needed.

**Pull request merged**